auth-web/src/main.ts

```diff
@@ -49,6 +50,14 @@ ConfigHelper.saveConfigToSessionStorage().then(async (data) => {
     })
   }
 
+  // initialize Hotjar
+  const hotjarId = ConfigHelper.getValue('HOTJAR_ID');
+  (<any>window).hotJarId = hotjarId
+  if (hotjarId) {
+    console.info('Initializing Hotjar...') // eslint-disable-line no-console
+    Vue.use(Hotjar, { id: hotjarId })
+  }
```

Comment:
The surveys are automatically displayed according to their configuration in Hotjar.

For the subject filing (IA Filing Survey)...

This type means it displays over top of the app:
![image](https://user-images.githubusercontent.com/10002889/214196845-2ce2f710-ec6a-447a-9326-b31e4543476f.png)

This targeting means it watched the URL params:
![image](https://user-images.githubusercontent.com/10002889/214196983-d3a5c6be-2fdc-4f90-8b12-dd0d61b69bf4.png)

In other words, if the URL is

Reply:
A side effect of simply enabling Hotjar is that recordings are automatically captured by default, which gives us trends, etc.

To reduce traffic, etc, I have disabled recordings in Hotjar for "dev.bcregistry.ca".

Should we also disable Test and Prod?

cc: @pwei1018